filterizePy/test/test_greenscale.py

```diff
@@ -0,0 +1,74 @@
+# Copyright 2019 Akansha Vashisth
+# Licensed under the MIT License (the "License").
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at https://mit-license.org
+
+# February 2019
+# This script tests color function of filterizePy package.
+import pytest
+import numpy as np
+import pandas as pd
+import os
+import imghdr
+import skimage.io
+from filterizePy.greenscale import greenscale
+
+def test_greenscale_white():
+    # White image testing
+    test_img = np.array([[[255, 255, 255],[255, 255, 255],[255, 255, 255]] ,
+                          [[255, 255, 255],[255, 255, 255],[255, 255, 255]],
+                          [[255, 255, 255],[255, 255, 255],[255, 255, 255]],
+                          [[255, 255, 255],[255, 255, 255],[255, 255, 255]],
+                          [[255, 255, 255],[255, 255, 255],[255, 255, 255]]], dtype = "uint8")
+    gs_test_img = np.array([[[  0, 230, 0],[  0, 230, 0],[  0, 230,  0]],
+                             [[  0, 230, 0],[  0, 230, 0],[  0, 230,  0]],
+                             [[  0, 230, 0],[  0, 230, 0],[  0, 230,  0]],
+                             [[  0, 230, 0],[  0, 230, 0],[  0, 230,  0]],
+                             [[  0, 230, 0],[  0, 230, 0],[  0, 230,  0]]], dtype = "uint8")
+
+    greenscale("../../img/test_img.png")
+    output = skimage.io.imread("../../img/gs_test_img.png")
+    assert np.array_equal(output, gs_test_img), "The greenscale function does not work properly."
+
+
+def test_greenscale_black():
+    # Black image testing
+    test_img_black = np.array([[[0, 0, 0]]], dtype = "uint8")
+    gs_test_img_black = np.array([[[0, 0, 0]]], dtype = "uint8")
+
+    greenscale("../../img/test_img_black.jpeg")
+    output = skimage.io.imread("../../img/test_img_black.jpeg")
```

Comment:
I would name it `expected_output` or something so to not be confused with the output from your green scale function

Reply:
Everything else looks good to me! Great work!

Reply:
Sure that makes it more clear. Thanks!